.github/workflows/npm-app-snapshot-release.yml

```diff
@@ -86,11 +91,12 @@ jobs:
         echo "NPM is sensitive to .npmrc formatting; check that it has a newline at the end!!!"
         npm ci --unsafe-perm
         npm run prod
-        # version=$(jq -r .version package.json)
+        version=$(jq -r .version package.json)
```

Comment:
Not sure why this was commented out, and what the consequence of putting it back is

Reply:
oh god. probs just because it was unused